pkg/resources/virtual/common/common.go

```diff
@@ -25,25 +24,29 @@ type DefaultFields struct {
 }
 
 // GetTransform produces the default transformation func
-func (d *DefaultFields) GetTransform() cache.TransformFunc {
-	return d.transform
+func (d *DefaultFields) GetTransform() func(obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	return d.TransformCommon
 }
```

Comment:
Nitpick: now that `TransformCommon` is public, there is effectively no difference between `GetTransform` and `TransformCommon`. I suggest removing one of the two.

Reply:
Yes, no need for 'GetTransform'